packages/hardhat/hardhat.config.ts

```diff
@@ -29,6 +29,7 @@ const EVM_PRIVATE_KEY_2 = vars.get(
   "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
 );
 const ETHERSCAN_API_KEY = vars.get("ETHERSCAN_API_KEY", "DNXJA8RX2Q3VZ4URQIWP7Z68CJXQZSC6AW");
+const BASE_SEPOLIA_API_KEY = vars.get("BASE_SEPOLIA_API_KEY", "E3X4WTYQKAWFW5DW4G8AAN4E761PSJH2N6");
```

Comment:
should this be in codebase?

Reply:
@masonicGIT I believe these are API keys that were generated with our personal accounts. They are not as dangerous to leak as something like the Alchemy keys, because they are from personal accounts and the process for deleting a key is very quick. However it is bad to be in the habit of checking-in these keys, because we should be building the habit of storing these separately from git.

Reply:
These are keys from our free accounts, so we're not leaking anything that can cause a security problem, nor a financial problem.